.github/workflows/golangci-lint.yaml

```diff
@@ -24,13 +24,12 @@ jobs:
     name: golangci-lint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/setup-go@v4
-        with:
-          go-version: "1.21"
-          cache: false
```

Comment:
It seems to me that `cache: false` here isn't necessary, but I'm not sure. @travisn ?

Reply:
Agreed, I don't see why we would need it. The flag is not used for any other actions.